stdlib/types.jou

```diff
@@ -0,0 +1,28 @@
+# Functions for checking types
```

Comment:
As discussed in your previous PR, I'm not yet sure if I want these in Jou's stdlib.

Reply:
In my issues #376
In the self-hosted file "token.jou"
There is a TODO:
```python
# TODO: move to stdlib
declare isprint(b: int) -> int
```

Reply:
just think it later

Reply:
Maybe add a `?` after the TODO:

```
# TODO: move to stdlib?
```

Reply:
Fine, but I think we can add it into stdlib.